Formula/n/neovide.rb

```diff
@@ -44,16 +44,14 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/neovide --version")
```

Comment:
not quite sure what happened with the CI, but running `--version` check separately works out fine.
![image](https://github.com/Homebrew/homebrew-core/assets/1580956/7645247d-bf6d-45f7-9db4-6f35f48846fc)

![image](https://github.com/Homebrew/homebrew-core/assets/1580956/f7320d94-49b3-4d3f-ad78-63b50ba3bc0c)